docs/index.md

```diff
@@ -1,4 +1,175 @@
+
+Queries to the database can be sent over HTTP through the Query Service API
+[Swagger Documentation](https://query.data.humancellatlas.org/v1/ui/#/)
+or via the [Query Builder](https://query.data.humancellatlas.org/).
+
+## Executing Queries
+To execute a query via the swagger interface: 
+- Click on the green `/query` row to expand it
+- Click `Try it out`
+- Edit the request body by adding in your query and any parameters. 
+
+For example
+```
+{
+  "params": {"s": 0},
+  "query": "select * from files where size > %(s)s limit 10"
+}
+```
+- click `Execute`
+
+Note: Queries executed via the swagger interface can not contain new lines or tabs (you will need to edit some of the 
+example queries by removing line breaks/tabs to make them work). These will generate a 400 error response 
+containing the line `"detail": "Request body is not valid JSON"`. For complex queries that benefit from multi-line 
+formatting we recommend using the [Query Builder](https://query.data.humancellatlas.org/). 
+## Async Queries
+For long-running queries (runtime over 20 seconds), the Query Service supports asynchronous tracking of query results.
+When a long-running query triggers this mode, the caller will receive a
+[`301 Moved Permanently`](https://en.wikipedia.org/wiki/HTTP_301) response status code with a `Retry-After` header. The caller
+is expected to wait the specified amount of time before checking the redirect destination, or use the query job ID
+returned in the response JSON body to check the status of the query job. The caller may turn off this functionality
+(and cause the API to time out and return an error when a long-running query is encountered) by setting the
+`async=False` flag when calling `/query`.
+
+For large query results, the Query Service may deposit results in S3 instead of returning them verbatim in the response
+body. In this case, the client will receive a [`302 Found`](https://en.wikipedia.org/wiki/HTTP_302) response status code
+sending them to the response data location. In this mode, response data are confidential to the caller, and remain
+accessible for 7 days. The caller may turn off this functionality by setting the `async=False` flag when calling
+`/query`.
+
+# Data Schema
+![](QueryServiceDataSchema.svg)
+
+Because there are often multiple, slightly different versions of a bundle or file, the `bundles_all_versions` and
+`files_all_versions` tables contain all versions of the bundles and files. There are also derived view tables `files` 
+and `bundles`, which only contain the latest version of each bundle or file. 
+
+The metadata itself is available in `files.body` as a JSON data object. The structure of that document is dependent on 
+the `dcp_schema_type_name` column. The schemas for each schema type can be found [here](https://schema.humancellatlas.org/a). 
+It is also possible to pull the full JSON document for a file of a particular schema type (or the `aggregat_metadata` 
```

Comment:
Typo and missing word here: "`aggregat_metadata`" -> "`aggregate_metadata` field"